services/filesstore/filesstore_test.go

```diff
@@ -308,3 +308,50 @@ func (s *FileBackendTestSuite) TestRemoveDirectory() {
 	_, err = s.backend.ReadFile("tests2/asdf")
 	s.Error(err)
 }
+
+func (s *FileBackendTestSuite) TestAppendFile() {
+	// First part needs to be at least 5MB for the S3 implementation to work.
+	size := 5 * 1024 * 1024
+	b := make([]byte, size)
+	for i := range b {
+		b[i] = 'A'
+	}
+	path := "tests/" + model.NewId()
+
+	written, err := s.backend.AppendFile(bytes.NewReader(b), path)
```

Comment:
nit: these cases could be split in `s.Run`s for clarity, like `s.Run("Should fail because there is no file to append to")`